### PR TITLE
[patch] remove cockroachdb/errors

### PR DESCRIPTION
**Pull request by kpango**
Signed-off-by: kpango <i.can.feel.gravity@gmail.com>

### Description:
remove cockroachdb/errors for license problem

### Environment:

- Go Version: 1.15.1
- Docker Version: 19.03.8
- Kubernetes Version: 1.18.2
- NGT Version: 1.12.1

### Types of changes:

- [x] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [x] Refactoring [type/refactoring]
- [x] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [x] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [x] I have added tests and benchmarks to cover my changes.
- [x] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by rinx — DISMISSED** [review]
LGTM

**Review by hlts2 — APPROVED** [review]
LGTM

**Comment by kpango** [comment]
@rinx @hlts2 I passed all test code and deep source.
I changed errors/errors.go's Is function & test comparator error message format.
Others no changes from yesterday you reviewed, would you mind if I merge this PR?

File: apis/swagger/agent/core/core/agent.swagger.json [hunk 1]
```diff
@@ -349,8 +349,7 @@
           "format": "int64"
         },
         "indexing": {
-          "type": "boolean",
-          "format": "boolean"
```
Comment (rinx) [inline apis/swagger/agent/core/core/agent.swagger.json:353]: why these swagger files updated?

Comment (kpango) [reply apis/swagger/agent/core/core/agent.swagger.json:353]: don't know, I just run make update for dependencies update.
may be swagger version ?

Comment (rinx) [reply apis/swagger/agent/core/core/agent.swagger.json:353]: okay

**Review by rinx — APPROVED** [review]

File: apis/swagger/agent/core/core/agent.swagger.json (hunk 1 above)
Comment (kpango) [reply apis/swagger/agent/core/core/agent.swagger.json:353]: I checked swagger specification, there is not format field for boolean type.
maybe old version has format for boolean type.
FYI: https://swagger.io/specification/#data-types

Comment (kpango) [reply apis/swagger/agent/core/core/agent.swagger.json:353]: @rinx (mention for FYI) and I'll merge this PR

**Pull request merged**